app/javascript/availability/libraries_locations.json

```diff
@@ -600,7 +607,7 @@
     "SLTXTBK": "Schuylkill - Textbooks - In-Library Use",
     "SMALL-MP": "Maps Library, Central Pattee, Basement - Small Map Collection",
     "SMITH": "Stored offsite. Ask at Special Collections, 104 Paterno",
-    "SOFTWAREHN": "Copy unavailable, request via Interlibrary Loan",
+    "SOFTWAREHN": "Hazleton - Software - Collection Temporarily Unavailable",
```

Comment:
same for this location, it is in the ill list and the label change here won't have an effect.

Reply:
now moved out of ILL list